Duplicati/Library/Encryption/GPGEncryption.cs

```diff
@@ -272,5 +273,14 @@ private void Runner(object x)
             Utility.Utility.CopyStream((Stream)tmp[0], (Stream)tmp[1]);
             ((Stream)tmp[1]).Close();
         }
+
+        /// <summary>
+        /// Determines the path to the GPG exe
+        /// </summary>
+        public static string GetGpgExePath()
+        {
+            var gpg4win = System.IO.Path.GetFullPath(@"C:\Program Files (x86)\GnuPG\bin\gpg.exe");

```

Comment:
It appears that this is likely the default install location, but can a user install GnuPG to a different path?  I'm not very familiar with Windows, but is there a way to ask the registry if this is installed and what the corresponding path is?

Reply:
Latest commit will use the registry to check for the install.